main-app/app/scripts/providers/model/game-model.js

```diff
@@ -1,124 +1,51 @@
-                                }
-
-                                if (data.outcome === 'Win') {
-                                    alert("A winner is Player " + data.winner + "!");
-                                    me.gameActive = false;
-                                }
-                                else if (data.outcome === 'Draw') {
-                                    alert("It's a tie!");
-                                    me.gameActive = false;
-                                }
-                            })
-                            .error(function (data, status, headers, config) {
-                                alert(data);
-                            });
+                    if (proxy.currentPlayer === 1) {
+                        return proxy.p1Name;
+                    }
+                    else if(proxy.currentPlayer === 2) {
+                        return proxy.p2Name;
                     }
                 };
 
-                var populateBoard = function(boardData) {
-
-                    me.board = boardData;
+                this.startNewGame = function() {
 
-                    for(var i=0;i<boardData.length;i++) {
+                    return proxy.startNewGame();
+                };
 
-                        switch(boardData[i]) {
-                            case '0':
-                                document.getElementsByClassName("boardSquare")[i].classList.remove("nought");
-                                document.getElementsByClassName("boardSquare")[i].classList.remove("cross");
-                                break;
-                            case '1':
-                                document.getElementsByClassName("boardSquare")[i].classList.add("cross");
-                                break;
-                            case '2':
-                                document.getElementsByClassName("boardSquare")[i].classList.add("nought");
-                                break;
-                        }
-                    }
+                this.makeMove = function(square) {
 
+                    return proxy.makeMove(square);
                 };
 
                 this.clearBoard = function() {
 
-                    populateBoard("000000000");
+                    return proxy.clearBoard();
                 };
 
                 this.go = function(path) {
 
-                    p1State = 'human';
-                    p2State = 'human';
+                    proxy.p1State = 'human';
+                    proxy.p2State = 'human';
                     $location.path(path);
                 };
 
                 this.goAndUpdatePlayers = function(path) {
 
-                    p1State = document.getElementsByClassName("playerState")[0].innerHTML.toLowerCase();
-                    p2State = document.getElementsByClassName("playerState")[1].innerHTML.toLowerCase();
+                    proxy.p1State = document.getElementsByClassName("playerState")[0].innerHTML.toLowerCase();
```

Comment:
This is something you will need to fix in the next iteration - you [almost] never need to access the document object: model values should be bound to the view and these accessed.